app/controllers/reviewable_claimed_topics_controller.rb

```diff
@@ -26,6 +31,26 @@ def destroy
       reviewable.log_history(:unclaimed, current_user)
     end
 
+    notify_users(topic, nil)
     render json: success_json
   end
+
+  private
+
+  def notify_users(topic, claimed_by)
+    user_ids = User.staff.pluck(:id)
```

Comment:
What about in the case of category moderation or when moderators cannot see it? We can't assume that only staff reviews content.

A reviewable has fields `reviewable_by_moderator` and `reviewable_by_group` to handle this.